### PR TITLE
Rollup nitpick round

### PR DESCRIPTION
**Pull request by ChihChengLiang**

### CODE REVIEW

**Review by vaibhavchellani — APPROVED** [review]
LGTM

File: test/rollup.test.ts [hunk 1]
```diff
@@ -118,7 +118,7 @@ describe("Rollup", async function() {
         const previousMP = initialBatch.proofCompressed(0);
         const commitmentMP = targetBatch.proof(0);
 
-        const _tx = await rollup.disputeBatch(
+        const _tx = await rollup.disputeTransition_Transfer(
```
Comment (vaibhavchellani) [inline test/rollup.test.ts:121]: Do we want to stick with camel case?

Comment (ChihChengLiang) [reply test/rollup.test.ts:121]: Hmmm, the original idea was to get the clarity by splitting the feature "Transfer" from the nature of the dispute, "a dispute of state transition".
But maybe following the mixedCase for function as style guide suggested for simplicity is better. Will fix this

Comment (ChihChengLiang) [reply test/rollup.test.ts]: Fixed

**Pull request merged**